### PR TITLE
Fix formatting errors

### PR DESCRIPTION
**Pull request by StephenGill**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -54,7 +54,8 @@ en:
         <li>the Ministry of Housing, Communities & Local Government</li>
         <li>the Department for Environment Food & Rural Affairs</li>
         <li>the Department for Work & Pensions</li>
-        <li>the National Health Service (NHS)</li> 
+        <li>the National Health Service (NHS)</li>
+        </ul>
```
Comment (issyl0) [inline config/locales/en.yml]:
```suggestion
      </ul>
```

**Review by leenagupte — APPROVED** [review]
👍 

screenshot-localhost_5000-2020 03 31-10_17_16

**Review by issyl0 — APPROVED** [review]
Tiny style thing but LGTM.

File: config/locales/en.yml (hunk 1 above)
Comment (leenagupte) [reply config/locales/en.yml]: good spot!

**Pull request merged**